kickstarts/base.ks.erb

```diff
@@ -325,6 +327,15 @@ ln -sf /dev/null /etc/systemd/system/ctrl-alt-del.target
 mkdir /var/log/journal
 systemctl restart systemd-journald
 
+# Alter cloud-init config to allow root ssh access to the appliance
+[[ -s /etc/cloud/cloud.cfg ]] && sed -i "s/^ssh_pwauth:.*$/ssh_pwauth: True/g" /etc/cloud/cloud.cfg
+[[ -s /etc/cloud/cloud.cfg ]] && sed -i "s/^disable_root:.*$/disable_root: false/g" /etc/cloud/cloud.cfg
```

Comment:
disable_root had no effect with the older version of cloud-init. Only the ssh_pwauth was needed. Nice to see it's working now!